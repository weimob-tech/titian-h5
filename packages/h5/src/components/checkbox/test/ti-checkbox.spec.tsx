import { newSpecPage } from '@stencil/core/testing';
import { TiCheckbox } from '..';

describe('ti-checkbox', () => {
  it('renders', async () => {
    const page = await newSpecPage({
      components: [TiCheckbox],
      html: `<ti-checkbox></ti-checkbox>`,
    });
    expect(page.root).toMatchSnapshot();
  });
});
