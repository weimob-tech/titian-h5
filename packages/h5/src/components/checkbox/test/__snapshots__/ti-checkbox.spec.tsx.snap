// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ti-checkbox renders 1`] = `
<ti-checkbox>
  <mock:shadow-root>
    <div class="titian-checkbox titian-checkbox-row">
      <div class="titian-checkbox-icon-wrap">
        <ti-icon ext-class="titian-checkbox-icon titian-checkbox-icon-zoom-out" name="checkbox-hollow"></ti-icon>
      </div>
      <div class="titian-checkbox-label">
        <slot></slot>
        <slot name="label"></slot>
      </div>
    </div>
  </mock:shadow-root>
</ti-checkbox>
`;
